packages/twenty-front/src/modules/command-menu/hooks/useMatchActions.ts

```diff
@@ -0,0 +1,40 @@
+import { ActionConfig } from '@/action-menu/actions/types/ActionConfig';
+import { getActionLabel } from '@/action-menu/utils/getActionLabel';
+import { isNonEmptyString } from '@sniptt/guards';
+import { useDebounce } from 'use-debounce';
+
+export const useMatchActions = ({
```

Comment:
Can we add more context in the naming ? It seems to be related to the search text in the command menu that filters the actions ? We should be able to understand without having to read the internal code of this hook.